added MIT license (#175)
HGVSnomenclature/hgvs-nomenclature: docs/index.md

## A note about our name

We've called ourselves by lots of names, including "VarNomen", "MutNomen", "HGVS
Recommendations", "HGVS Guidelines", and even just "HGVS" colloquially. We
recognize that this has caused confusion in the community. Please call the
specification "HGVS Nomenclature".

## Availability

> This repository contains both text content and code. The licensing for each is as follows:
> 
> * The text content of this repository is released under the [CC0 1.0 Universal (CC0 1.0) Public Domain Dedication](https://creativecommons.org/publicdomain/zero/1.0/). See the [`LICENSE-CC0.txt`](LICENSE-CC0.txt) file for details.
> * The code, configuration files, and other computer-readable artefacts in this repository are released under the [MIT License](https://opensource.org/licenses/MIT). See the [`LICENSE.txt`](LICENSE.txt) file for details.

Reviewer: I hope I'm not too picky; since this text is rendered on the website and it's mostly aimed at non-technical users, I think we should replace "repository" with "website" and remove the MIT license info. E.g.:

```suggestion
The contents of this website are released under the [CC0 1.0 Universal (CC0 1.0) Public Domain Dedication](https://creativecommons.org/publicdomain/zero/1.0/). See the [`LICENSE-CC0.txt`](LICENSE-CC0.txt) file for details.
```

The rest looks good to me!

Author: @ifokkema "Code, configuration files, and other computer-readable artefacts" is intended to cover the grammar in the yaml. It's nascent now, but I hope we extend that grammar so that the repo and website are grammar-first -- we make changes there, then document in the website. I think that grammar needs a license to make usage clear.

Does an explanation of my intention change your suggestion?

Author: (So, to be clear, I'd like to keep the MIT license primarily for the grammar)

Reviewer: Ah, OK, so how about:

```suggestion
This website contains both text content (the documentation) and code (the HGVS nomenclature structure). The licensing for each is as follows:

* The text content of this website is released under the [CC0 1.0 Universal (CC0 1.0) Public Domain Dedication](https://creativecommons.org/publicdomain/zero/1.0/). See the [`LICENSE-CC0.txt`](LICENSE-CC0.txt) file for details.
* The code and configuration files of this website are released under the [MIT License](https://opensource.org/licenses/MIT). See the [`LICENSE.txt`](LICENSE.txt) file for details.
```

I'm mostly worried about confusing users with technical terms like "repository" when they see this on the website.

Reviewer 2: I understand the need for some additional clarification here, but wanted to back away from "website" references and instead describe the HGVS Recommendations as such rendered as .pdf. I think the below may clarify:

```suggestion
The [code repository](https://github.com/HGVSnomenclature/hgvs-nomenclature/pull/175) from which the HGVS Nomenclature is rendered contains both written text recommendations and associated code. The licensing for each is as follows:

* The rendered text of the HGVS Nomenclature is released under the [CC0 1.0 Universal (CC0 1.0) Public Domain Dedication](https://creativecommons.org/publicdomain/zero/1.0/). See the [`LICENSE-CC0.txt`](LICENSE-CC0.txt) file for details.
* The code, configuration files, and other computer-readable artefacts in the code repository are released under the [MIT License](https://opensource.org/licenses/MIT). See the [`LICENSE.txt`](LICENSE.txt) file for details.
```
